### ISSUE TITLE
Add term noctural pain

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10488]
**Preferred term label:**
Noctural pain

**Synonyms**
Night pain

**Definition (free text, please give PubMed ID)**
(increased) pain at night PMID37214788

**Parent term (use hpo.jax.org/app)**
Pain characteristic HP:0025280

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Osteoarthritis (omim165720) PMID37214788

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
@lisannesw 
Adding new term: 
- ID: HP:6000707
- label: Noctural pain
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: PMID:37214788
- def: Pain at night.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10488
- synonyms:
